### PR TITLE
Proposal for managing instructor training workflow

### PR DESCRIPTION
**Pull request by gvwilson**

### CODE REVIEW

**Comment by pbanaszkiewicz** [comment]
@gvwilson do you want me to take it from here or just add my comments?

**Comment by gvwilson** [comment]
Add comments and I'll rewrite - seems only fair I do *some* work :-)

File: 2016/ideas-list-swc.md [hunk 1]
```diff
@@ -231,14 +232,100 @@ Other: please use our [mailing list][].
...
+| Moderate | Workflow Design, Interface Design, Python, Django, Testing | [@pbanaszkiewicz][] |
+
+### Background
+
+[Software Carpentry][swc] has built a Django-based web application called [AMY][]
+to keep track of who's qualified to teach,
+what workshops have run,
+who taught and attended,
+and so on.
+This project will extend it to manage all aspects of instructor training,
```
Comment (pbanaszkiewicz) [inline 2016/ideas-list-swc.md]: Add one paragraph about what instructor training is.

File: 2016/ideas-list-swc.md [hunk 2]
```diff
@@ -231,14 +232,100 @@ Other: please use our [mailing list][].
...
+
+**Please ask questions as issues [here][gsoc-issues].**
+
+### Abstract
+
+Extend [AMY][] to manage the process of training people to be instructors.
+
+| **Intensity** | **Involves**  | **Mentors** |
+| ------------- | --------------|------------ |
+| Moderate | Workflow Design, Interface Design, Python, Django, Testing | [@pbanaszkiewicz][] |
```
Comment (pbanaszkiewicz) [inline 2016/ideas-list-swc.md]: I can mentor the technical part, but you, @gvwilson, should be here as a consulting mentor – you know the great deal of training process and you know your specific requirements.

File: 2016/ideas-list-swc.md [hunk 3]
```diff
@@ -231,14 +232,100 @@ Other: please use our [mailing list][].
...
+(which relies on a lot of email, some Etherpads, and a Google Docs spreadsheet)
+to determine what we're actually doing;
+translating that workflow into new pages in [AMY][]
+that can be used by administrators, trainers, and mentors;
+making that workflow's implementation flexible so that it can easily be reconfigured as our process evolves;
+testing both the usability and functionality of the new interface.
+
+The software stack for this project is [Python][], [Django][], and [JQuery][].
+
+## Open Source Development Experience
```
Comment (pbanaszkiewicz) [inline 2016/ideas-list-swc.md]: I don't know how that header works. Is the proposal supposed to be divided in two parts, @rgaiacs? I know this pattern repeats in other proposals too.

I'd change this header to `###`.

File: 2016/ideas-list-swc.md [hunk 4]
```diff
@@ -231,14 +232,100 @@ Other: please use our [mailing list][].
...
+
+Skills required to complete the project (you can apply without all of them):
+
+* Knowledge of [Python][].
+* Knowledge of [Django][] or a similar three-tier web application framework.
+* Knowledge of [JQuery][] or a similar front-end Javascript UI toolkit.
```
Comment (pbanaszkiewicz) [inline 2016/ideas-list-swc.md]: @gvwilson how do you expect to use jQuery in this project?

**Comment by pbanaszkiewicz** [comment]
@gvwilson I think that's all my comments.

**Comment by pbanaszkiewicz** [comment]
Looks okay, thanks Greg.

**Pull request merged**